Update README.md

gxmlfx opened the pull request:
fix an wrong command
docker compose -d docker-compose.rocm.yml up --> docker compose -f docker-compose.rocm.yml up -d

matatonic commented:
Oh, thank you! For completeness, can you add the -d to all the compose up commands?

gxmlfx commented:
> Oh, thank you! For completeness, can you add the -d to all the compose up commands?

it's not missing an '-d', it's on the wrong position

gxmlfx commented:
> Oh, thank you! For completeness, can you add the -d to all the compose up commands?

actually this command missing an '-f', which is necessary because there is a yml file, that's actually why this command is wrong and would call error

gxmlfx commented:
That's crazy, I recommand you this wrong command, you closed my pr and mock me and then you fix it?

matatonic commented:
-d was a good suggestion, I've been busy with other things and updated it with another change.
No mockery intended - sorry you got the wrong idea!

Thanks for pointing out the error.

gxmlfx commented:
> -d was a good suggestion, I've been busy with other things and updated it with another change.
> 
> No mockery intended - sorry you got the wrong idea!
> 
> 
> 
> Thanks for pointing out the error.

I never suggested you to add an -d, the real wrong is this command missing a -f， and I added it

matatonic commented:
ok
